webui/master/src/containers/pages/Metrics/Metrics.tsx

```diff
@@ -290,6 +291,35 @@ export class MetricsPresenter extends React.Component<AllProps> {
             </Table>
           </div>
         ))}
+        {Object.keys(data.journalDiskMetrics).map((key: string) => (
```

Comment:
Since `journalDiskMetrics` is an array, it might not be necessary to use `map` here since you're already creating the table rows starting line 307. Mapping here will cause a new table w/ header "Alluxio Master Journal Disk Status" to be created for each array item. You want to just create one table w/ rows mapped from the array right?

Reply:
I think it would be better design to only have one "journal disk warning" and the rest of the rows not have the header. I was aware of the problem you pointed out before but I was a little lazy to make it better. I'll spend some time to implement a version which doesn't repeat the Message

Reply:
Here's a screenshot after the latest commit (e20c835):
![image](https://user-images.githubusercontent.com/3464508/113674258-db046780-966e-11eb-91a5-2918078eb01c.png)

Reply:
![](https://user-images.githubusercontent.com/3464508/112932855-8479b500-90d3-11eb-839d-3e7e9f259310.png)

The `Alluxio Master Journal Disk Status` table (including table headers) will be rendered for each item in `data.journalDiskMetrics`. did you want it this way? or just to show a new table row per item?

Reply:
Oh wait...my previous comment I was referring to the overview page. For some reason I didn't realize this comment was on the metrics page 🤦 I will fix this part tonight as well....

Reply:
ook.. should be fixed now